common/scala/src/main/scala/whisk/http/ErrorResponse.scala

```diff
@@ -114,8 +114,8 @@ object Messages {
     /** Error for meta api. */
     val propertyNotFound = "Response does not include requested property."
     def invalidMedia(m: MediaType) = s"Response is not valid '${m.value}'."
-    val contentTypeNotSupported = """Content type must be specified and one of ["json", "html", "http", "text"]."""
-
+    val contentTypeExtentionNotSupported = """Content type must be specified and one of ["json", "html", "http", "text"]."""
```

Comment:
while you're at it - mind changing ` contentTypeExtentionNotSupported = """Content type must` to ` contentTypeExtentionNotSupported = """Extension must`?

maybe also prefix each of the extensions with a `.` (dot).